weddineo-frontend/apps/weddineo-frontend/src/assets/i18n/pl_PL.json

```diff
@@ -6,5 +6,9 @@
       "password": "Hasło",
       "signIn": "Zaloguj się "
     }
+  },
+  "footer": {
+    "version": "wersja",
+    "revision": "commit"
```

Comment:
a think there is no need to display 'commit' istead of revision.
Other think is if I have named it correctly, but if we want to change it, model variables names should be also changed.

Reply:
Ok. I will change label to revision